src/Tools/BuildBoss/.gitignore

```diff
@@ -0,0 +1 @@
+launchSettings.json
```

Comment:
I want to F5 BuildBoss on my local machine and doing so drops a launchSettings.json file into my source tree. That necessarily hard codes all of the paths that I put into the Debug page for the project. No reason to check that in.

So either i need to remember to reset that file every single time for I commit.  Or blanket ignore it so that I don't have to think about it when i do a `git -A`. 

The launchSettings.json experience is painful.